demo/Titanic/README.md

```diff
@@ -45,6 +45,17 @@ To run the demo execute the `run_demo.py` script in either of the two available
 
 If you choose the `hit` scenario, don't be surprised if you encounter some failures along the route...
 
+### Step 4: Running the demo game (optional)
+
+The demo also contains a playable version of the software under test. To start the game, follow the following instructions
+
+* Install matplotlib: `pip install matplotlib`
+* Based on your system, install curses
+  * for Windows: `pip install windows-curses`
+  * for UNIX based systems it should work out of the box
+* Navigate to your local `demo/Titanic` folder
+* Run: `python run_game.py`
```

Comment:
At first this doesn't appear to work for a novice user. When loading the game the focus switches to the game window, but here the controls do not work. You have to switch back to the command window to see the instructions and control the game.